llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -732,17 +773,25 @@ def tensor_hook(
                         monitor_process = None
 
                     # Spawn a new process to register the model.
+                    # Slower method to register the model via log_model.
                     process = SpawnProcess(
-                        target=_register_model_with_run_id_multiprocess,
+                        target=_log_model_multiprocess,
                         kwargs={
                             'mlflow_logger':
                                 mlflow_logger,
-                            'composer_logging_level':
-                                logging.getLogger('composer').level,
-                            'model_uri':
-                                local_save_path,
-                            'name':
+                            'transformers_model_path':
+                                temp_save_dir,
+                            'flavor':
+                                'peft' if self.using_peft else 'transformers',
+                            'python_logging_level':
+                                logging.getLogger('llmfoundry').level,
+                            'task':
```

Comment:
we should just be ** the `self.mlflow_logging_config`, rather than picking out pieces of it.

Reply:
Other functions were flattening the fields, so I thought that was best practice. I can pass in the full object, which I agree is cleaner.